### PR TITLE
Add ombi

### PR DESCRIPTION
**Pull request by amjed-ali-k**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by githubsaturn** [comment]
Thanks!

File: public/v4/apps/ombi.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        restart: unless-stopped
+        volumes:
+            - $$cap_appname-config:/config
+        caproverExtra:
+            containerHttpPort: '3579'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_ombi_version
+          label: Ombi Version
+          defaultValue: development-v4.0.1446-ls143
```
Comment (githubsaturn) [inline public/v4/apps/ombi.yml:18]: Usually it's better to go with stable versions, like [version-v4.0.497](https://hub.docker.com/layers/linuxserver/ombi/version-v4.0.497/images/sha256-97e801a46a4ddec5ddbdc2ce4f07502e00e60ce6e9e275a0868628344f597106?context=explore)

Comment (amjed-ali-k) [reply public/v4/apps/ombi.yml:18]: This one doesn't have a stable version for now.

Comment (githubsaturn) [reply public/v4/apps/ombi.yml:18]: `version-v4.0.497` isn't stable?

Comment (amjed-ali-k) [reply public/v4/apps/ombi.yml:18]: Yes, it is. I didn't found that one. lol

**Pull request merged**